docs/jwst/resample/arguments.rst

```diff
@@ -15,35 +15,61 @@ image.
     image.  Available kernels are `square`, `gaussian`, `point`,
     `turbo`, `lanczos2`, and `lanczos3`.
 
+    For spectral data, only the `square` and `point` kernels should be used.
+
 ``--pixel_scale_ratio`` (float, default=1.0)
-    Ratio of input to output pixel scale.  A value of 0.5 means the output
+    Ratio of input to output pixel scale.
+
+    For imaging data, a value of 0.5 means the output
     image would have 4 pixels sampling each input pixel.
+
+    For spectral data, values greater than 1 indicate that the input
+    pixels have a larger spatial scale, so more output pixels will
+    sample the same input pixel.  For example, a value of 2.0
+    means the output image would have 2 pixels sampling each input
+    spatial pixel. Note that this parameter is only applied in the
+    cross-dispersion direction: sampling wavelengths are not affected.
+
```

Comment:
Is it also reminding users what effect this will have on the flux in each pixel?   E.g. if the pixel_scale is 2.0, the pixel values in the _s2d file will be approximately halved.

Reply:
Good idea.  I added a comment to that effect, for both pixel_scale_ratio and pixel_scale.